grunt/rollup-conf.js

```diff
@@ -32,6 +34,8 @@ module.exports = {
             banner: outputParts[0],
             footer: outputParts[1],
             plugins: [
```

Comment:
I thought about having 2 builds and using some pre-processing etc. But it seemed fugly.


What about exposing method for setting protobuf implementation from openapi-clientlib user space? 
To support protbuf, we would import protobuf implementation on **go** side and pass it to openapi-clientlib. 


If protobuf will not be passed, we will force standard, JSON serialization method.


Are there any other openapi-client lib users that want to benefit from protobuf, or go is main and only userspace atm? 

We already require external dependencies ie. SignalR and Jquery. So precedence is there. However, in this case it would be configurable optional external dependency. 

I think this allows for more flexibility. In future we could detect passed protobuf implementation and support few of them. 

Example usage

```
import protobuf from 'protobufjs';
...
streamingOptions = {
    transportTypes: ['longPolling'],
    serializationFormats: [
         'application/x-protobuf': protobuf,
    ],
}
...
openApi.streaming = new openapi.Streaming(retryTransport, openApiConfig.streamingUrl, transportAuth.auth, streamingOptions)
```

Reply:
As discussed, added API for defining external serializers and underlying engines like protobufjs.

In addition to defining serializer per format, user is able to set underlying engine.